src/pages/Checkout/CheckoutBetaPage.vue

```diff
@@ -53,14 +55,17 @@ export default {
 			currentStep: 'basket',
 			loans: [],
 			totals: () => {},
-			donations: []
+			donations: [],
+			loading: false,
 		};
 	},
 	apollo: {
 		query: initializeCheckout,
-		prefetch: true,
-		result({ data }) {
-			console.log(data);
+		preFetch: true,
```

Comment:
@BoulderBrains I had originally put 'prefect' instead of 'preFetch' so now we're getting the server render...

Reply:
👍